base/scripts/code.antiscum.cs

```diff
@@ -43,17 +43,15 @@
     return;
   }
 
-  if (%force) {
-    if (%timeLeft < 0) {
-      %timeLeft = 0;
-    }
-    Client::sendMessage(%cl, 1, "You have " @ %timeLeft @ " seconds to bring the flag home.~wshell_click.wav");
-  }
-
+  if (%timeLeft < 0) { %timeLeft = 0; }
+  
   if (%timeLeft > 5) {
-    if (%timeLeft == 30 || %timeLeft == 15 || %timeLeft == 10) {
+      
+    if (%timeLeft == 30 || %timeLeft == 20 || %timeLeft == 10 || %force) {
```

Comment:
hey, this still isn't equivalent to before. the previous behavior would send the message as soon as someone left the safe zone. i think this is important for the player to know exactly when their timer begins. the current implementation doesn't tell them when they leave the safe zone.

is your goal here to avoid the double message? if so, you can put back the original if-statement block and change this logic to check for `!%force`. that way the two blocks can't both occur.